src/functions/create-timed-curation/handler.ts

```diff
@@ -51,6 +51,24 @@ const createTimedCuration: ValidatedEventAPIGatewayProxyEvent<typeof schema> = a
     }
   }
 
+  if (promoted.length === 1 && documents[0].data?.meta_content_category === ContentCategory.MANUAL) {
```

Comment:
Remember to update this after removing the type from the category field and adding a separate boolean field for manual documents.

Reply:
Updated